src/modules/account/views/MyProfile.tsx

```diff
@@ -259,6 +291,19 @@ export const AccountMyProfile: NextPage<DefaultSeoInfo> = ({ url }) => {
 		</FormControl>
 	);
 
+	const renderLanguagePreferencesForm = () => (
```

Comment:
None of this component seems to interact with anything else in `MyProfile`, move it to its own component for better separation of concerns.

Reply:
isn't that a bit overkill for such a small component?